weave

```diff
@@ -950,7 +1001,7 @@ case "$COMMAND" in
         CONTAINER=$(docker start $CONTAINER_ID)
         get_ipam_cidr $CONTAINER_ID $CONTAINER
         with_container_netns $CONTAINER_ID attach $CIDR_ARGS >/dev/null
-        tell_dns PUT $CONTAINER_ID $CIDR_ARGS
+        when_dns_running when_has_fqdn $CONTAINER put_dns_fqdn $CIDR_ARGS
```

Comment:
The `with_container_netns` call ends up setting `$CONTAINER` to `$CONTAINER_ID`, so this is not doing what you want.

Simple fix is to call `with_container_netns` with `$CONTAINER` instead of `$CONTAINER_ID`.